logs_analyzer/chapter2/java8/src/main/java/com/databricks/apps/logs/LogAnalyzerRDD.java

```diff
@@ -12,23 +12,23 @@
 import org.apache.spark.sql.SparkSession;
 
 public class LogAnalyzerRDD {
-  private final SparkSession sparkSession;
+  private final SparkSession spark;
 
-  public LogAnalyzerRDD(SparkSession sparkSession) {
-    this.sparkSession = sparkSession;
+  public LogAnalyzerRDD(SparkSession spark) {
```

Comment:
Procedure syntax deprecated. Return Unit. Also, the enclosing braces are not required